plugins/Goals/DataTable/Filter/CalculateConversionPageRate.php

```diff
@@ -100,7 +100,7 @@ private function getGoalTotalConversions(DataTable $table, array $goalIds): arra
 
         foreach ($goalIds as $idGoal => $g) {
             $total = $archive->getNumeric(GoalsArchiver::getRecordName('nb_conversions', $idGoal));
-            if (count($total)) {
+            if (!empty($total) && count($total)) {
                 $goalTotals[$idGoal] = reset($total);
             }
```

Comment:
`$archive->getNumeric` might either return an array or a single value, depending on the parameters the archive was requested with. So guess something like this should cover all cases correctly.

```suggestion
            if (is_array($total) && count($total)) {
                $goalTotals[$idGoal] = reset($total);
            } else if (is_numeric($total)) {
                $goalTotals[$idGoal] = $total;
            }
```

Reply:
Updated